pawankrgithub/task2.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title> Retro City</title>
+    <style>
+        .right{
+        height: 100% ;
+        width: 100%;
+        }
+        .bottom{
+         background-color: rgb(253, 194, 122);
+        box-shadow: 10px 10px rgb(193, 45, 185);
+        position:absolute;
+        bottom:2%;
+        right:2%;
+        width:150px;
+        height:80px;
+        padding:3px;
+
+        }
+    </style>
+</head>
+<body>
+<div class="bottom">
```

Comment:
this div box isn't relative to the image. I suggest you to add this box and image inside a div that has `position: relative;`